docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -13,31 +13,53 @@ in Playwright-specific methods, for example, `ignoreHTTPSErrors`, `extraHTTPHead
 [[synthetics-config-file]]
 
 Create a `synthetics.config.js` or `synthetics.config.ts` file in the root of the
-synthetics project and specify the options:
+synthetics project and specify the options. For example:
 
-[source,js]
+[source,ts]
 ----
-import { SyntheticsConfig } from '@elastic/synthetics'
-
-const config: SyntheticsConfig = {
-  params: {
-    url: 'https://www.elastic.co'
-  },
-  playwrightOptions: {
-    ignoreHTTPSErrors: true, // ignores all HTTPS errors during navigation
-    extraHTTPHeaders: {
-      'foo': 'bar' // additional HTTP headers to be sent with every request
-    }
-  },
-  monitor: {
-    schedule: 10,
-    locations: [ 'us-east4-a' ],
+import type { SyntheticsConfig } from '@elastic/synthetics';
+
+export default env => {
+  const config: SyntheticsConfig = {
+    params: {
+      url: 'https://www.elastic.co',
+    },
+    playwrightOptions: {
+      ignoreHTTPSErrors: false,
+    },
+    /**
+     * Configure global monitor settings
+     */
+    monitor: {
+      schedule: 10,
+      locations: [ 'us-east4-a' ],
```

Comment:
nit - I know this was like this before, but I'd suggest changing it to a _real_ value such as `us_east`